src/components/Modals/SignOutModal.jsx

```diff
@@ -1,50 +1,51 @@
-import React from 'react'
-import Modal from 'react-modal'
-import firebase from '../../config/firebase'
-import { connect } from 'react-redux'
-import { toggleSignOutModal, logInStateChange } from '../../actions/actionCreators'
+import React from 'react';
+import Modal from 'react-modal';
+import firebase from '../../config/firebase';
+import { connect } from 'react-redux';
+import { toggleSignOutModal, logInStateChange } from '../../actions/actionCreators';
 
-Modal.setAppElement('#root')
+Modal.setAppElement('#root');
 
-class SignOutModal extends React.Component {
-  signOut = () => {
-    firebase.auth().signOut()
-    localStorage.removeItem('userInfo')
-    this.props.dispatch(logInStateChange({ userUID: null, userLoggedIn: false }))
-    this.props.dispatch(toggleSignOutModal())
-  }
+const SignOutModal = ({
+  isOpen,
+  logInStateChange,
+  toggleSignOutModal,
```

Comment:
thanks. here we would need `dispatch` as a prop. We can remove `logInStateChange` and `toggleSignOutModal` as they are not props but actions from line 5. I recommend trying out [React dev tools](https://chrome.google.com/webstore/detail/react-developer-tools/fmkadmapgofadopljbjfkapdkoienihi?hl=en) if you havent. its great for looking up the props and state in a component.